accounting/account/management/commands/generate_account_data.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# pylint: disable=unused-variable,no-self-use
+"""
+Management command for generating data (usually for a development environment) to play with.
+"""
+
+import logging
+
+from django.core.management import BaseCommand
+from django.db import IntegrityError, transaction
+
+from accounting.account.tests import factories
+from accounting.bank.tests.factories import BankAccountFactory
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    A management command to generate data for the Account application.
+    """
+
+    help = 'Generate data for the Account app, usually to play with in a development environment.'
+
+    DEFAULT_ACCOUNTS_NUM = 10
+    DEFAULT_PASSWORD = 'password'  # NOQA
+    DEFAULT_IS_STAFF = False
+    DEFAULT_IS_SUPERUSER = False
+    DEFAULT_BANK_ACCOUNTS = True
+
+    def add_arguments(self, parser):
+        """
+        Entry point for custom arguments.
+        """
+        parser.add_argument(
+            '--accounts-num',
+            action='store',
+            dest='accounts_num',
+            type=int,
+            default=self.DEFAULT_ACCOUNTS_NUM,
+            help='The number of accounts to generate, including address and hourly rate data. Use an even number, '
+                 'since we use the HourlyRate model to generate accounts, which generates 2 accounts per instance. '
```

Comment:
I should say 'HourlyRate *factory*' rather than 'model' here, because that's what's actually happening: the HourlyRate factory generates a client and provider account. So if you ask for 3 accounts, it'll generate 2 (3 // 2 = 1, leading to 1 `HourlyRate` object, and thus 2 accounts).

I have adopted the suggestion too though.